modules/php/Models/Scoresheets/Scoresheet3.php

```diff
@@ -94,6 +95,18 @@ public function getScribbleReactions(Scribble $scribble): array
       Notifications::circleAntennas($this->player, $scribbles);
     }
 
+    if ($scribble->getNumber() === NUMBER_X) {
```

Comment:
Ok i understand the factorisation here but we might want to keep track where the X is comming from. Probably useless for now but one of the campaign card is adding a new construction cards with an X and it shouldnt be confused with the X placed from planning (you can see other funky cards in the spritesheet).
So maybe an additional parameter to getScribbleReaction with the function that called it (chooseCards / systemError / writeX)

Reply:
That's already in place, isn't it?

Reply:
Yeah i added it in the meantime